zqd/api/api.go

```diff
@@ -200,6 +199,11 @@ func (ps *PacketSearch) FromQuery(v url.Values) error {
 	}
 	ps.Span = span
 	ps.Proto = v.Get("proto")
+	switch ps.Proto {
+	case "tcp", "udp", "icmp":
+	default:
```

Comment:
Nit: `if p := ps.Proto; p != "tcp" && p != "udp" && p != "icmp" {` is one line instead of three.

Reply:
They're both not great.

Reply:
True!